frontend/src/pull-card/refresh.tsx

```diff
@@ -0,0 +1,51 @@
+import { Pull } from '../pull';
+import { refreshPull } from '../backend/pull-socket';
+import { Box, chakra } from "@chakra-ui/react"
+import { useState } from "react"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faRefresh } from '@fortawesome/free-solid-svg-icons'
+
+const RefreshContainer = chakra(Box, {
+   baseStyle: {
+      lineHeight: 0.9,
+      visibility: "hidden",
+      position: "absolute",
+      cursor: 'pointer',
+      bottom: '3px',
+      right: '3px',
+      borderRadius: '4px',
+      borderWidth: '1px',
+      borderColor: 'var(--refresh)',
+      padding: "5px",
+      background: "white",
```

Comment:
Since we are merging #264 into this branch, does this mean that this pull is blocked until #264 gets merged since #264 will override some of these settings? 

https://github.com/iFixit/pulldasher/blob/d1adef54ac2d5781e37569e1219f7c14270d2194/frontend/src/pull-card/refresh.tsx#L20

Reply:
Other way around... That one should kinda be blocked on this one. Either way, it shouldn't matter that much, either can be merged first, it'll just make this diff larger though.